src/git_changelog/build.py

```diff
@@ -459,7 +459,7 @@ def _assign_previous_versions(self) -> None:
                         previous_semver = semver
                         previous_version = next_commit.tag
                 else:
-                    next_commits = next_commit.parent_commits + next_commits
+                    next_commits.extend(next_commit.parent_commits)
```

Comment:
Just a small change, the comparison of the semver makes it unnecessary to add the parent commits at the beginning of the list. I did it, because I thought it would be better to first follow the left branch of the graph.

Reply:
I see, that looks good, thanks :+1: